SU-HSDO/suhumsci
# ARCH-305 updated page template and search styling

# READY FOR REVIEW

# Summary
- Updated page template to removed decanter grid classes from header and search region
- Added necessary styling so that search behaves properly at all breakpoints

# Needed By (Date)
- End of sprint (8/2)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that search is no longer floating all over the place, but rather has sensible positioning

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-305

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
